### PR TITLE
Goodbye sidebar! (Styles to make this work)

### PR DESCRIPTION
**Pull request by thisisdano**
These style help regularize the widths of dashboard elements if there's no sidebar.

**Paired with https://github.com/18F/cg-dashboard/pull/893**
**Based off Icon/Panel work in https://github.com/18F/cg-dashboard/pull/892 and https://github.com/18F/cg-style/pull/250 — Merge them first.**

- - -

screen shot 2016-12-30 at 7 27 00 am

- - -

screen shot 2016-12-30 at 7 27 18 am

### CODE REVIEW

File: src/css/components/header.scss [hunk 1]
```diff
@@ -10,12 +10,18 @@
   @include clearfix;
   margin: 0 auto;
   max-width: $site-max-width;
-  padding-left: 1rem;
-  padding-right: 1rem;
+  padding-left: $grid-2;
+  padding-right: $grid-2;
```
Comment (thisisdano) [inline src/css/components/header.scss:15]: converted to grid scale

File: src/css/components/header.scss [hunk 2]
```diff
@@ -10,12 +10,18 @@
...
   padding-top: $grid-5;
   padding-bottom: $grid-3;
 }
 
+.header-dashboard {
+  .header-wrap {
+    max-width: $dashboard-max-width;
+  }
+}
+
```
Comment (thisisdano) [inline src/css/components/header.scss:24]: added a dashboard-specific max width

File: src/css/components/main-content.scss [hunk 3]
```diff
@@ -3,11 +3,32 @@
...
+    margin: 0;
+    max-width: none;
+    padding: $grid-4 0;
+    width: auto;
+  }
+
+  .grid {
+    max-width: none;
+  }
+}
```
Comment (thisisdano) [inline src/css/components/main-content.scss:34]: dashboard-specific styling. set a common max-width, as well as consistent padding

File: src/css/override_components.scss [hunk 4]
```diff
@@ -54,6 +54,13 @@
   }
 }
 
+.disclaimer-dashboard {
+  .grid {
+    max-width: $dashboard-max-width;
+    padding: 0 $grid-2;
+  }
+}
+
```
Comment (thisisdano) [inline src/css/override_components.scss:63]: set a dashboard-specific max-width

File: src/css/override_vars.scss [hunk 5]
```diff
@@ -97,10 +97,11 @@ $grid-n4:             #{($grid-base * -4 / $grid-type)}rem;     // 32px
 $grid-n5:             #{($grid-base * -5 / $grid-type)}rem;     // 40px
 
 // widths
-$nav-width:          $grid-base-px * 75;          // 600px;
-$site-max-width:     $grid-base-px * 130;         // 1040px;
-$text-max-width:     36em;
-$text-narrow-width:  17em;
+$nav-width:             $grid-base-px * 75;          // 600px;
+$site-max-width:        $grid-base-px * 130;         // 1040px;
+$dashboard-max-width:   $grid-base-px * 111;         // 888px;
```
Comment (thisisdano) [inline src/css/override_vars.scss]: And here's that dashboard-specific max-width var

**Review by thisisdano — COMMENTED** [review]
My comments on my changes

**Comment by msecret** [comment]
I think this needs a rebase, it has some changes from https://github.com/18F/cg-style/pull/250/ which makes it hard to parse.

**Comment by thisisdano** [comment]
Let's just wait for 250 and do that first, as I said in the top note. Then it'll be clearer.

**Comment by msecret** [comment]
250 is merged already though?

**Comment by thisisdano** [comment]
Sorry @msecret — my error. Should be ready to go now.

File: src/css/components/header.scss (hunk 2 above)
Comment (msecret) [reply src/css/components/header.scss:24]: Can we not make it dashboard specific but just something like "large"?

File: src/css/components/main-content.scss [hunk 6]
```diff
@@ -11,3 +11,24 @@
...
+
+.main_content {
+  &.content-dashboard {
+    margin: 0 auto;
+    max-width: $dashboard-max-width;
+    padding: 0 $grid-2;
+  }
+}
+
+.content-dashboard {
```
Comment (msecret) [inline src/css/components/main-content.scss]: Same as above.

File: src/css/override_components.scss (hunk 4 above)
Comment (msecret) [reply src/css/override_components.scss:63]: Same as above

**Review by msecret — COMMENTED** [review]
Only thing, I wanted to keep "dashboard" out of the naming as we could then use these styles on a site that isn't the dashboard too. I was thinking "large" but there could be a better name out there. Maybe something related to the site not having a sidenav. I'll brainstorm:

- `no_sidenav`
- `full`
- `complete`
- `solo`
- `single`

File: src/css/components/header.scss (hunk 2 above)
Comment (thisisdano) [reply src/css/components/header.scss:24]: well, it's not large, it's actually narrower and it _is_ specific to the dashboard

**Comment by thisisdano** [comment]
I could change to something like `no_sidenav`, but I guess I don't see the reason for forward compatibility with a page that doesn't exist and has no plans to exist — and this is specifically tooled to the dashboard. but really I don't care and just want to see it merged, so I'll change.

**Comment by thisisdano** [comment]
@msecret
OK, made that change here and in the `cg-dashboard` pair.

File: src/css/override_vars.scss [hunk 7]
```diff
@@ -99,7 +99,7 @@ $grid-n5:             #{($grid-base * -5 / $grid-type)}rem;     // 40px
 // widths
 $nav-width:             $grid-base-px * 75;          // 600px;
 $site-max-width:        $grid-base-px * 130;         // 1040px;
-$dashboard-max-width:   $grid-base-px * 111;         // 888px;
+$dashboard-no_sidebar:   $grid-base-px * 111;         // 888px;
```
Comment (msecret) [inline src/css/override_vars.scss]: Shouldn't this be `$no_sidebar-max_width` now?

**Comment by thisisdano** [comment]
@msecret
Oops, yup. Changed to `$no-sidebar-max-width`

**Review by msecret — APPROVED** [review]
